app/_locales/en/messages.json

```diff
@@ -1673,7 +1673,7 @@
     "message": "Suggest address names"
   },
   "externalNameSourcesSettingDescription": {
-    "message": "We pull data from third parties like Etherscan, Infura, and Lens Protocol, to suggest names for addresses on signatures requests. Turning on name suggestions exposes your IP address to these third parties."
+    "message": "We pull data from third parties like Etherscan, Infura, and Lens Protocol, to suggest names for addresses on signature or approval requests. Turning on name suggestions exposes your IP address to these third parties."
```

Comment:
We typically have any long form text like this reviewed by Barbara who will pass it to a technical writer.

My vote would be to reference `signature and transaction requests`.

Reply:
cc @bschorchit can you take a look?

Reply:
Same ask for this @coreyjanssen 🙏